packages/ifpack2/src/Ifpack2_ILUT_def.hpp

```diff
@@ -152,16 +152,16 @@ void ILUT<MatrixType>::setParameters (const Teuchos::ParameterList& params)
   using Teuchos::Exceptions::InvalidParameterType;
 
   // Default values of the various parameters.
-  int fillLevel = 1;
+  double fillLevel = 1.0;
   magnitude_type absThresh = STM::zero ();
   magnitude_type relThresh = STM::one ();
   magnitude_type relaxValue = STM::zero ();
   magnitude_type dropTol = ilutDefaultDropTolerance<scalar_type> ();
 
   bool gotFillLevel = false;
   try {
```

Comment:
For later (not this PR): Let's get rid of the "try-catch as control logic" and replace with `params.isType<double>("fact: ilut level-of-fill")` etc.